### PR TITLE
2007 - Updates Placeholder Links

### LINKED ISSUE #2007: Sidebar placeholder links to low-level TinaCMS toolkit field docs
## Description

After you add Tina to your site with our CLI init, run things, and navigate to one of the site's main pages, you will see:
Screen Shot 2021-08-06 at 12 04 07 PM

This links to our TinaCMS Toolkit [fields docs](https://tina.io/docs/fields/). I would think it instead makes sense to now link to our schema docs. Maybe [this](https://tina.io/guides/tina-cloud/add-tinacms-to-existing-site/content-modelling/)?

### Steps to reproduce

Add tinacms to a site, and navigate to a page without any form registered

### Expected result

Link to Tina Cloud content modeling docs

### Actual result

Links to TinaCMS low level field docs

#### Tinacms Package Versions

├─┬ @tinacms/cli@0.53.0
│ ├─┬ @tinacms/graphql@0.51.1
└─┬ tinacms@0.52.0
  ├─┬ @tinacms/toolkit@0.50.1

### PR DESCRIPTION
**Pull request by Enigmatical**
James accurately pointed out that the link for the `NoFormsPlaceholder` was now not as relevant.
Updated to point to our page on Content Modelling.

Closes #2007

### CODE REVIEW

**Review by DirtyF — DISMISSED** [review]

**Comment by DirtyF** [comment]
What if you don't use Tina Cloud?

File: packages/@tinacms/toolkit/src/packages/react-sidebar/components/NoFormsPlaceHolder.tsx [hunk 1]
```diff
@@ -31,11 +31,8 @@ export const NoFormsPlaceholder = () => (
       so you can start editing.
     </p>
     <p>
-      <LinkButton
-        href="https://tina.io/docs/forms/"
-        target="_blank"
-      >
-        <Emoji>📖</Emoji> Form Setup Guide
+      <LinkButton href="https://tina.io/docs/schema/" target="_blank">
+        <Emoji>📖</Emoji> Content Modelling
```
Comment (jeffsee55) [inline packages/@tinacms/toolkit/src/packages/react-sidebar/components/NoFormsPlaceHolder.tsx:35]:
```suggestion
        <Emoji>📖</Emoji> Content Modeling
```
Not sure 🤔 https://www.grammarly.com/blog/modeling-or-modelling/

**Review by jeffsee55 — APPROVED** [review]

File: packages/@tinacms/toolkit/src/packages/react-sidebar/components/NoFormsPlaceHolder.tsx (hunk 1 above)
Comment (Enigmatical) [reply packages/@tinacms/toolkit/src/packages/react-sidebar/components/NoFormsPlaceHolder.tsx:35]: We've got a few changes to make if we're going with `Modeling` than `Modelling`...

**Comment by Enigmatical** [comment]
> What if you don't use Tina Cloud? what if you're an editor just clicking on the pencil icon?

A good question that I don't have the answer for.  The link is based off James' suggestion.  
What would you propose?  This is in `tinacms`, so is there a universal location?

**Review by logan-anderson — APPROVED** [review]

**Pull request merged**